workflow/pred_temperature_ridge.ipynb

```diff
@@ -24,11 +22,12 @@
     "- Train-test split (70%/30%)\n",
     "- Preprocessing based on the training set (`s2spy`)\n",
     "- Resample data to the calendar (`lilio`)\n",
-    "- Dimensionality reduction and model training, with cross-validation (`lilio` & `scikit-learn`)"
+    "- Dimensionality reduction and cross-validation (`lilio` & `scikit-learn`)\n",
+    "- Training and testing the model\n",
+    "- Saving the trained model with `onnx`"
```

Comment:
Could be nice to turn this list into hyperlinks. Also make it a numbered list where the numbers align with the numbered headers?